packages/workers-shared/asset-worker/tests/kv.test.ts

```diff
@@ -66,9 +62,7 @@ describe("[Asset Worker] Fetching assets from KV", () => {
 
 			await expect(() =>
 				getAssetWithMetadataFromKV(mockKVNamespace, "abcd", undefined, 2)
-			).rejects.toThrowError(
-				"Requested asset abcd could not be fetched from KV namespace."
-			);
+			).rejects.toThrowError("KV GET abcd failed.");
```

Comment:
we should probably reject with an error in one of these to test the message is correctly injected